production/helm/loki/values.yaml

```diff
@@ -359,7 +359,7 @@ loki:
   commonConfig:
     path_prefix: /var/loki
     replication_factor: 3
-    compactor_address: '{{ include "loki.compactorAddress" . }}'
+    compactor_grpc_address: '{{ include "loki.compactorAddress" . }}'
```

Comment:
Add a comment here that compactor_grpc_address is prefered over compactor_address. If a customized compactor_address is set, compactor_grpc_address must be set to an empty string.